examples/demo-dts/src/config/app.config.ts

```diff
@@ -11,40 +12,40 @@ export default registerAs('appConfig', () => ({
    * Defaults to 5000 if APP_PORT is not set in the environment variables.
    * @type {number}
    */
-  appPort: parseInt(process.env.AGENT_PORT, 5000) || 5000,
+  appPort: parseInt(process.env.AGENT_PORT) || 5000,
 
   /**
    * Hostname or IP address for the PostgreSQL database.
-   * Defaults to an empty string if POSTGRES_HOST is not set in the environment variables.
+   * Defaults 'postgres' string if POSTGRES_HOST is not set in the environment variables.
    * @type {string}
    */
-  postgresHost: process.env.POSTGRES_HOST || '',
+  postgresHost: process.env.POSTGRES_HOST || 'postgres',
```

Comment:
I understand that for Unic.ID Issuer DTS we may want to force using Postgres backend, but do you think we need to force it also in the demo DTS?

In my opinion, it is desirable to keep the number of required components to the minimum. Of course, it is good to have these postgres-related variables because it is good to facilitate the creation of more robust DTS based on this example.

Reply:
IMHO,  it would be better to keep the PostgreSQL configuration as it is currently recommended for our ecosystem's functionality. While it's true that making it configurable for future use is a good idea, I believe it would be worth leaving it as an issue to address later when the need arises to adapt to additional databases.

Reply:
I was actually referring to leaving it blank like we do in Service Agent to use embedded SQLite database instead of an external one. This way, we can set up a lightweight development demo composed by only two pieces: Service Agent and chatbot DTS backend (i.e. no need of postgres and redis).

In this case, I think this can be achieved by allowing the specification of a database type (`type` parameter in TypeORM), so a `databaseType` environment variable can be added. I guess this will require to add SQLite driver as a dependency for the demo.

But no problem on leaving it for another PR.